schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/CommonInterpreted.avdl

```diff
@@ -208,8 +208,9 @@ protocol CommonInterpreted {
 
     /**
     The consequence type for a given variant as defined by the Sequence Ontology (SO) (e.g.: id = SO:0001816 ; name = non synonymous)
+    NOTE: the record name is in plural to avoid a collision with OpenCB's ConsequenceType
     */
-    record ConsequenceType {
+    record ConsequenceTypes {
```

Comment:
should this be plural ? You can only have one `id` and one `name` right ?

Reply:
you have indeed a good eye! There is a naming collision between models, so due to another record in a different package being called `ConsequenceType` and my lack of imagination I renamed it to `ConsequenceTypes`... I don't like it either

Reply:
Could we call this one `CommonConsequenceType` and the other one in the other package `OtherPackageConsequenceType` ?

Reply:
The other package is out of our hands as it is OpenCB's. I like `VariantConsequence` following Ensembl https://www.ensembl.org/info/genome/variation/predicted_data.html

Reply:
yeah @priesgo I definitely vote for `VariantConsequence`